pipeline-scripts/buildlib.groovy

```diff
@@ -1567,14 +1565,14 @@ def build_plashets(doozerOpts, version, release, dryRun = false) {
         // Populate plashets_built
         plashets_built = readYaml(file: "$working_dir/plashets_built.yaml")
         echo "plashets_built: $plashets_built"
-        if (major > 4 || major == 4 && minor >= 6) { // 4.6+ already fully migrated; no need to create repos on rcm-guest.
+        if (major > 4 || major == 4 && minor >= 6) { // 4.6+ already fully migrated; no need to create repos on ocp-artifacts.
             return plashets_built
         }
-        // Continue to build and sync plashet repos out to rcm-guest until RHCOS starts consuming content from ocp-artifacts.
+        // Continue to build and sync plashet repos out to ocp-artifacts until RHCOS starts consuming content from ocp-artifacts.
         plashets_built = [:]
     }
 
-    // produce plashet repos on rcm-guest
+    // produce plashet repos on ocp-artifacts
```

Comment:
The code before this line is for producing plashet repos on ocp-artifacts, while the code after this line is for producing plashet repos on rcm-guest. A replace for host name here is not valid.

Since 4.1-4.5 has EOL, I think we can remove the legacy code after this line.

Reply:
Fixed in 2af7e5965cfdfc480ffebb7e902032150ee109f4